src/tooling/piral-cli/src/common/config.ts

```diff
@@ -41,6 +42,22 @@ export interface PiralCliConfig {
    * Sets the schema version to be used for pilets.
    */
   schemaVersion?: PiletSchemaVersion;
+  /**
+   * Automatically open the browser.
+   */
+  open?: boolean;
```

Comment:
I'd recommend renaming it to `openBrowser`.

Reply:
Done.